src/main/resources/test_files/runtime-spark/DataFrames/GroupbyClauseError1.iq

```diff
@@ -1,4 +1,4 @@
-(:JIQS: ShouldCrash; ErrorCode="XPTY0004" :)
+(:JIQS: ShouldRun; Output="({ "guess" : "Bulgarian", "target" : "Albanian", "country" : 123, "choices" : [ "Albanian", "Bulgarian", "Russian", "Ukrainian" ], "sample" : "00b85faa8b878a14f8781be334deb137", "date" : "2013-08-19" }, { "guess" : "Bulgarian", "target" : "Albanian", "country" : "AU", "choices" : [ "Albanian", "Bulgarian", "Russian", "Ukrainian" ], "sample" : "00b85faa8b878a14f8781be334deb137", "date" : "2013-08-19" })" :)
```

Comment:
I think we should rename the file to GroupbyClause8.iq as it's not expecting an error any more. Also extending this test with a few more diverse fields would be good for completeness